sale_invoice_policy/models/sale_order_line.py

```diff
@@ -16,8 +16,7 @@ class SaleOrderLine(models.Model):
     )
     def _compute_qty_to_invoice(self):
         other_lines = self.filtered(
-            lambda l: l.product_id.type == "service"
-            or not l.order_id.invoice_policy
+            lambda l: not l.order_id.invoice_policy
```

Comment:
From reading previous PRs adding this condition, it seems like ignoring services so that they're handled with their product config is intended behaviour. Mostly likely because depending on your setup, you might be using `service_policy` from `sale_project` for services and it's easier to just let the product handle it.
I think it'll be a hard sell to get this merged, if you need it I suggest extracting the condition to a function and modifying it in your personal repo.

In any case you would also need to adapt `_compute_untaxed_amount_to_invoice` as well